src/components/modules/table-content/hooks/use-active-toc.ts

```diff
@@ -12,6 +12,7 @@ function useActiveToc() {
           // Remove active class from all toc items
           document.querySelectorAll('#table-of-content-headings a').forEach(e => e.classList.remove('active'));
           tocHeadingEl.classList.add('active');
+          tocHeadingEl.scrollIntoView({ behavior: 'smooth', block: 'nearest', inline: 'start' });
```

Comment:
Great job @RBoraurito, Could you check when you return again to the top of the page? The TOC is a bit off. Check the video:

https://github.com/bejamas/httptoolkit-website/assets/22168189/9cf9e96a-11c2-4d1f-9847-fa8b9526e819